### PR TITLE
chore: makefile cleanup/fixes

### PR DESCRIPTION
**Pull request by crodriguezvega**
## Description

In collaboration with @chatton!

closes: #2440 
closes: #1943 
closes: #8

---

Before we can merge this PR, please make sure that all the following items have been
checked off. If any of the checklist items are not applicable, please leave them but
write a little note why.

- [x] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/ibc-go/blob/master/CONTRIBUTING.md#pr-targeting))
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/main/docs/docs/building-modules/10-structure.md).
- [ ] Wrote unit and integration [tests](https://github.com/cosmos/ibc-go/blob/master/CONTRIBUTING.md#testing)
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [ ] Added relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [x] Re-reviewed `Files changed` in the Github PR explorer
- [ ] Review `Codecov Report` in the comment section below once CI passes

### CODE REVIEW

File: testing/simapp/simd/cmd/root.go [hunk 1]
```diff
@@ -164,7 +165,7 @@ func initRootCmd(rootCmd *cobra.Command, encodingConfig params.EncodingConfig) {
 		genutilcli.ValidateGenesisCmd(simapp.ModuleBasics),
 		AddGenesisAccountCmd(simapp.DefaultNodeHome),
 		tmcli.NewCompletionCmd(rootCmd, true),
-		testnetCmd(simapp.ModuleBasics, banktypes.GenesisBalancesIterator{}),
+		cmd.NewTestnetCmd(simapp.ModuleBasics, banktypes.GenesisBalancesIterator{}),
```
Comment (crodriguezvega) [inline testing/simapp/simd/cmd/root.go:168]: I removed our testnet.go file and import instead the one from the SDK. This allows us to run a 4-validator local testnet like is described [here](https://docs.cosmos.network/v0.45/run-node/run-testnet.html).

File: contrib/devtools/Makefile [hunk 2]
```diff
@@ -0,0 +1,76 @@
+###
```
Comment (crodriguezvega) [inline contrib/devtools/Makefile:1]: This file is needed to run some of the simulation tests.

File: contrib/test_cover.sh [hunk 3]
```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
```
Comment (crodriguezvega) [inline contrib/test_cover.sh:1]: This file is needed to run test coverage.

File: testing/simapp/simd/cmd/testnet.go [hunk 4]
```diff
@@ -1,397 +0,0 @@
-package cmd
```
Comment (colin-axner) [inline testing/simapp/simd/cmd/testnet.go:1]: wahoo!!

**Review by colin-axner — APPROVED** [review]
Yay! Thanks @crodriguezvega! I think this means we can run sims now

**Review by damiannolan — APPROVED** [review]

**Pull request merged**